README.md

```diff
@@ -290,9 +290,9 @@ end
 
 You'll sometimes see this I'm-going-to-assume-I-have-a-thing-that-works approach referred to as "top-down" programming. For an awesome video demonstration take a look at [this](https://vimeo.com/131588133) when you have a moment. The opposite approach would be "bottom-up" where we start with the smallest piece that we can get to work and start building on it. Both are totally valid.
 
-In order to follow MVC conventions, we're going to create this file in our `app/models` directory. Go ahead and `touch app/models/task.rb`
+In order to follow MVC conventions, we're going to create this file in our `app/models` directory. Let's create our new model using `touch app/models/task.rb`
 
-In that file, let's go ahead and create a `#new` method that returns a new Task object.
+In that file, create a `#new` method that returns a new Task object.
```

Comment:
Don't we want them to create the `#initialize` that's shown below? I might be missing something here though...